docs/user/supported_devices.rst

```diff
@@ -324,6 +324,7 @@ ramips-mt7621
 
 * NETGEAR
 
+  - EX6150
```

Comment:
Should we explicitly call that v1?

Reply:
"v1" is not mentioned anywhere and only "EX6150" is written on the device itself.
So i would say, no.

Reply:
i would say yes, as it would avoid confusion with the v2

Reply:
I dont want to be a dealbreaker. If you want to add the "v1", go ahead. :)

Reply:
done

Reply:
The situation with Netgear is a bit "problematic", as the v1 is not explicitly labled on the device. For the v2 it's not printed on the Box (v1 and v2 can be distinguished on the outside only by the MU-MIMO claim on the packaging) and on the device it's part of the model name.

So it's a compromise in some direction, regardless of the labling. Just wanted to share this insight here.